sap/5.1/modules/ROOT/pages/sap-connector-config-snc.adoc

```diff
@@ -0,0 +1,170 @@
+. Click the download button at the bottom of the screen and enter the target path for the download.
+. Click the import icon and enter the target path to upload your certificate.
+. Click *Add to Certificate List* and then click the save icon.
+. Open a cmd or terminal window and use the `sapgenpse` tool to link your certificate to your PSE:
++
+`sapgenpse maintain_pk -v -a <name_of_SAP_Server_Certificate.crt> -p <name_of_your_PSE.pse>`
++
+The following example links a certificate named `IDE.crt` to a PSE named `RFC.pse`:
++
+image::sap-connector-import-certificate.png[Link `IDE.crt` to `RFC.pse`]
+
+=== Authorize the SNC Client in SAP
+
+Add a new entry to the `VSNCSYSACL` view to enable your distinguished name to use SNC:
+
+. In the SAP GUI, navigate to the *SM30* transaction and enter `VSNCSYSACL` as the view name.
+. Click *Maintain*.
+. Select *E* as the work area.
+. In the *System ID* field, enter the SAP ID.
+. In the *SNC name* field, enter your distinguished name:
++
+`p:<your-distinguished-name>`
++
+. Select all of the checkboxes to activate the entries.
+. Click the save icon.
++
+* If the server accepts the SNC name, it displays a `Canonical Name Determined` message in the SNC data box.
++
+* If the server does not accept the SNC name, check that the name used on the PSE file is the same as the name used on the Mule app and the name listed on STRUST.
++
+. Enable the SAP user for the Mule app so the app can use the SNC connection type:
+	.. Navigate to the *SM30* transaction, specify the `VUSREXTID` view, and click *Maintain*.
+	.. In the *External ID type* field, select `DN` as the work area.
+	.. In the *External ID* field, enter the distinguished name.
+	.. In the *User* field, enter the SAP username.
+	.. Select *Activated*.
+	.. Click the save icon.
+
+== Configure SAP Connector
+
+Configure SAP Connector to enable SNC:
+
+. Access Anypoint Studio.
+. Select *SAP* in the Studio canvas.
+. Click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
+. Configure the connection type.
++
+For a simple connection, follow these steps:
+
+.. In the *Connection* field, select `Simple connection provider`.
+.. On the *General* tab, configure these fields:
+* *Username*
+* *Password*
+* *SAP system number*
+* *SAP client ID*
+* *Application server host*
+.. On the *Advanced* tab, configure extended properties.
+
++
+For an X.509 certificate connection, follow these steps:
+. In the *Connection* field, select `Certificate`.
+. On the *General* tab, configure these fields:
+** *X.509 Certificate* (not base64 encoded)
+** *Sap client ID*
+** *Application server host*.
+.. On the *Advanced* tab, configure the extended properties.
+
+=== SAP Server Side Extended Properties Example:
+
+The following table shows example values for the server side extended properties:
+
```

Comment:
We really should not be using tables for item+description or item+value, per the style reference.

Reply:
We don't agree. Connector docs use tables and we want to be consistent. This is a bigger discussion.